src/app/components/navigable-story-teaser/navigable-story-teaser.component.ts

```diff
@@ -46,8 +47,8 @@ import { RouterLink } from '@angular/router';
 	`,
 })
 export class NavigableStoryTeaserComponent {
-	story = input.required<StoryTeaser>();
-	selected = input<boolean>();
-	authorSlug = input.required<string>();
+	@Input() story!: StoryTeaser;
```

Comment:
These changes shouldn't be here. Please do a rollback to the changes in this file and submit it in the same branch.

Reply:
Alright, sure.